### DISCUSSION TITLE
醫療AI商品化！雲象骨髓抹片分類計數AI拿歐盟、臺灣醫材認證，下一步拚美FDA認證

### DISCUSSION THREAD

**Discussion by vvvioletta** [1101-GS4719-C/Python-Community-GS4719-C#173] [科技新知]
AI輔助醫師診斷已是趨勢，甚至在醫療難度極高的血液腫瘤項目上也能協助診斷。上週雲象與臺大醫院聯合開發的這套AI可辨識15種骨髓血液細胞，還能將原本耗時20分鐘的人工判讀時間，縮短為5分鐘。更聯手建立了世界上最大的骨髓抹片影像標註資料集，這些標註影像能作為教學之用，讓新進醫師更懂得判斷不同類型細胞和疾病，作為經驗傳承。

[醫療AI商品化！雲象骨髓抹片分類計數AI拿歐盟、臺灣醫材認證，下一步拚美FDA認證](https://www.ithome.com.tw/news/148026)

**Comment by Jie2222** [comment]
相對於其他領域的AI創新，醫療AI相對困難許多，非常需要政府以及市場的長期支持，更有賴於發展有效率的跨產業合作

**Reply by HSU-YU-JEN** [reply]
感覺這種困難不僅來自於人工技術數位化，也因為樣本取得不易，所以讓這個領域的AI創新成長較緩

**Reply by enoch-lightning** [reply]
我覺得確實如此。AI的技術，包括分類、迴歸、電腦視覺、機器學習與深度學習都已發展成熟。如血液腫瘤這種要由病患身上取得的資料，臺灣的病人可能也只有幾萬個，甚至幾千個。如果要使用機器學習，或深度學習訓練模型，並加以診斷病人血液腫瘤的好壞，資料(data)數應該不夠。

**Reply by enoch-lightning** [reply]
如果不使用機器學習與深度學習，單純用分類分兩類、三類、四類、五類或更多類，或是用迴歸找出血液中的什麼化學物質，與腫瘤發展的關係，藉以判斷腫瘤的好壞，幾萬筆或幾千筆的資料可能做得到。

**Reply by enoch-lightning** [reply]
而且，還有延伸的問題，就是有沒有人種的差異。亞洲人、非洲人、歐洲人、北美洲人與中南美洲人在基因上本來就有顯著的差異，就算是同一種疾病，身體的表現與反應也可能差很多。

**Reply by 107602505** [reply]
AI在醫療上面的開發確實還有些挑戰需要克服，雖然它可以提高臨床試驗效率降低成本，但病患的招募與選擇、患者對藥物的反應等等因素都會影響機器學習的成果，要讓人們願意使用AI開發的新藥物感覺還有一段路要走。

**Comment by MAKERRRofKEVIN** [comment]
之前有耳聞國內對於這類技術的法規都是相對嚴格的，很高興能看到此技術拿到醫材認證，背後代表的技術的純熟以及審核單位的接受度提高對產業和民眾皆是一大福音。

**Comment by 109802503gloria** [comment]
去年12月初去了台灣醫療科技展，大多醫院和科技公司都在發展AI相關技術，特別是在圖像辨識上，在我看來是已經在醫療界發展起來了，如果大家有興趣可以去參觀這個展覽，聽說每年的12月前後都有喔。
